web/src/main/java/se/inera/intyg/webcert/web/csintegration/aggregate/CreateDraftCertificateAggregator.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2024 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.webcert.web.csintegration.aggregate;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import se.inera.intyg.infra.security.common.model.IntygUser;
+import se.inera.intyg.webcert.web.csintegration.certificate.CreateDraftCertificateFromCS;
+import se.inera.intyg.webcert.web.csintegration.util.CertificateServiceProfile;
+import se.inera.intyg.webcert.web.integration.interactions.createdraftcertificate.v3.CreateDraftCertificateFromWC;
+import se.riv.clinicalprocess.healthcond.certificate.createdraftcertificateresponder.v3.CreateDraftCertificateResponseType;
+import se.riv.clinicalprocess.healthcond.certificate.createdraftcertificateresponder.v3.Intyg;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class CreateDraftCertificateAggregator {
```

Comment:
Do we not want to/can follow the pattern where the aggregator and the services implement the same interface?

Reply:
I think its possible to implement the interface ```CreateDraftCertificateResponderInterface```, but I decided to not do it in the first iteration because I thought it felt "forced". 

Mainly because we decided to leave alot of the common validation in ```CreateDraftCertificateResponderImpl```. And if we decided to implement the interface we would end up with four implementation, wich felt wrong to me. 

But I might be wrong and we should indeed implement the interface.

Reply:
I was more thinking of creating an interface like "CreateDraftCertificiate" to keep the pattern we've used in previous aggregator implementations, and make sure that the cs and wc implementation doesn't start to diverge. But maybe it's not the way to go

Reply:
Ah I see! I did not think of that

I like the suggestion and will add it to the PR!